api/route.proto

```diff
@@ -61,7 +62,7 @@ message Route {
   // Nginx vars used to do the route match.
   repeated Var vars = 9;
   // Embedded plugins.
-  google.protobuf.Any plugins = 10;
+  Plugins plugins = 10;
```

Comment:
Just for sure: as we use the concrete type to describe plugin, so once we introduce another plugin, we have to update the `plugins.proto` right?

Reply:
Yes.